### PR TITLE
Add Launchpad GCC ARM Embedded v2015q4

### PR DESCRIPTION
**Pull request by kaidokert**
Submitting this as per Homebrew/homebrew#45236

### CODE REVIEW

File: Casks/gcc-arm-embedded.rb [hunk 1]
```diff
@@ -0,0 +1,37 @@
+cask 'gcc-arm-embedded' do
+  version '5_2-2015q4'
+  sha256 '41056ffeba4bcb5bbea13185461a1269613ac13321fbda3e7dc59ee664ee3f06'
+
+  url "https://launchpad.net/gcc-arm-embedded/5.0/5-2015-q4-major/+download/gcc-arm-none-eabi-#{version}-20151219-mac.tar.bz2"
```
Comment (vitorgalvao) [inline Casks/gcc-arm-embedded.rb]: It has a date, so we’ll need to refine `version` to include it.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in acec844b84c2507dd0e071f8ef4b52b77f7f5f47. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Comment by kaidokert** [comment]
Thank you. Should the version specific part in URL somehow also be squashed ?
'5.0/5-2015-q4-major'

**Comment by vitorgalvao** [comment]
Ideally, but it starts being a complex mess, so lets way for the next update.

**Pull request closed without merging**